src/Compilers/CSharp/Test/Emit/CSharpCompilerEmitTest.csproj

```diff
@@ -55,9 +56,6 @@
     <Reference Include="..\..\..\..\..\packages\System.Reflection.Metadata.$(SystemReflectionMetadataVersion)\lib\portable-net45+win8\System.Reflection.Metadata.dll" />
     <Reference Include="..\..\..\..\..\packages\System.Collections.Immutable.$(SystemCollectionsImmutableVersion)\lib\portable-net45+win8+wp8+wpa81\System.Collections.Immutable.dll" />
     <Reference Include="..\..\..\..\..\packages\Microsoft.DiaSymReader.1.0.5\lib\net45\Microsoft.DiaSymReader.dll" />
-    <Reference Include="xunit">
-      <HintPath>..\..\..\..\..\packages\xunit.1.9.2\lib\net20\xunit.dll</HintPath>
-    </Reference>
```

Comment:
Also, currently we have to update at least 104 files just to change xUnit. This change will cut that down to 54 (53 packages.config files and 1 *.targets file).

Ideally, I would think we should be able to update xUnit with only 2 file changes (1 packages.config and 1 *.targets file). This would be possible since all of our UnitTests share a common base framework (and all of our packages go to the same output directory). We should then only need the xUnit package references in the base assembly (which would be synced and built as a dependency by the other 52 projects).